lib/ember-test-helpers/test-module.js

```diff
@@ -296,6 +296,9 @@ export default Klass.extend({
 
     if (this.registry) {
       this.registry.fallback.resolver = function() {};
+      if (hasEmberVersion(2,3)) {
+        this.registry.fallback.resolver.resolve = function() {};
+      }
```

Comment:
Might be _slightly_ better as:

```
      if (hasEmberVersion(2,3)) {
        this.registry.fallback.resolver = {
          resolve: function() {};
        };
      } else {
        this.registry.fallback.resolver = function() {};
      }
```

Will consider whether this can be handled even more elegantly.